### PR TITLE
deleted old unit test that are part of new unit tests

### PR DESCRIPTION
**Pull request by vikash018**
deleted VerticaPy/verticapy/tests/vDataFrame/test_vDF_correlation.py old unit test as chaid, pivot_table_chi2 test functions covered under new_test VerticaPy/verticapy/tests_new/core/vdataframe/test_machine_learning.py

deleted test_vDF_combine_join_sort.py old unit test as these tests are covered under test_join_union_sort.py, test_machine_learning.py, test_pivot.py and test_agg.py
old test | new test reference
test_vDF_append - test_join_union_sort.py
testvDF_groupby - test_agg.py
test_vDF_join- test_join_union_sort.py
test_vDF_narrow- test_pivot.py
test_vDF_pivot - test_pivot.py
testvDF_polynomial_comb - test_machine_learning.py
testvDF_recommend - test_machine_learning.py
test_vDF_sort- test_join_union_sort.py

deleted test_vDF_sessionize function from old test test_vDF_feature_engineering.py as this function is implemented under test_machine_learning.py

deleted following function under test_vDF_preprocessing.py old unit test as these are covered under new_test
old test | new test reference
test_vDF_train_test_split - test_machine_learning.py
test_vDF_add_duplicates - test_machine_learning.py
test_vDF_cdt - test_machine_learning.py
test_vDF_cut - test_encoding.py
test_vDF_decode - test_encoding.py
test_vDF_discretize - test_encoding.py
test_vDF_merge_similar_names - test_pivot.py
test_vDF_mean_encode - test_encoding.py
test_vDF_fillna - test_fill.py
test_vDF_clip - test_fill.py
test_vDF_fill_outliers - test_fill.py
test_vDF_scale - test_normalize.py
test_vDF_outliers - test_machine_learning.py
test_vDF_bool_to_int - test_typing.py

### CODE REVIEW

File: contributing/README.md [hunk 1]
```diff
@@ -136,8 +136,8 @@ tox -e py36 -- verticapy/tests
 tox -e py37 -v -- verticapy/tests
 
 # Run an individual test on specified python versions.
-# e.g.: Run the test `test_vDF_append` under `test_vDF_combine_join_sort.py` on the python versions 3.7 and 3.8
-tox -e py37,py38 -- verticapy/tests/vDataFrame/test_vDF_combine_join_sort.py::TestvDFCombineJoinSort::test_vDF_append
+# e.g.: Run the test `test_append` under `test_join_union_sort.py` on the python versions 3.7 and 3.8
+tox -e py37,py38 -- verticapy/tests_new/core/vdataframe/test_join_union_sort.py::TestJoinUnionSort::test_append
```
Comment (oualib) [inline contributing/README.md:140]: change python version pls. We support 39 or greater.

**Review by oualib — CHANGES_REQUESTED** [review]
Please, change the Python version in the contributing and triple check if you did not erase anything by mistake.

@mail4umar can you please quickly check this PR?

File: contributing/README.md (hunk 1 above)
Comment (vikash018) [reply contributing/README.md]: I have updated python version to 39 or greater in README.me

**Comment by vikash018** [comment]
I have double checked on old deleted unit test, and we are only deleting duplicate tests in old unit tests as its implemented in new unit test.

**Pull request merged**